AutomatedTesting/Gem/PythonTests/AWS/Windows/aws_metrics/aws_metrics_automation_test.py

```diff
@@ -3,24 +3,38 @@
 For complete copyright and license terms please see the LICENSE at the root of this distribution.
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Prerequisites for running AWS automation tests on your local Windows machine:
+1) Go to the AWS IAM console and create an IAM role called o3de-automation-tests with admin permissions
+2) Grant your AWS account permission to assume the o3de-automation-tests IAM role following:

```

Comment:
When run these tests locally, users need to assume a role under their own AWS account. Should follow the same steps as cross account use case, otherwise the default AWS identity is not able to assume the role.